Xamarin.Forms.Platform.iOS/ViewRenderer.cs

```diff
@@ -36,12 +36,38 @@ public abstract class ViewRenderer<TView, TNativeView> : VisualElementRenderer<T
 #endif
 		NativeColor _defaultColor;
 
+		event EventHandler<PropertyChangedEventArgs> _elementPropertyChanged;
```

Comment:
Do we really need this can we just use the override? Why ?

Reply:
This is the event that the *Managers* hook into

https://github.com/xamarin/Xamarin.Forms/pull/1974/files#diff-f7da8c7c17ec3976ea074bebce797504R12

This way one manager can just hook into this event and then apply the effects of that change back to the same control. This is why this PR removes a lot of the ElementPropertyChanged watching into the managers so the behavior can be generalized